front/app/components/PageForm/index.tsx

```diff
@@ -1,56 +1,22 @@
 import React from 'react';
```

Comment:
Refactored this component by 
- Creating a sub-component for each field (these live in `PageForm/fields`)
- Dividing parts of the `validatePageForm` up in functions that live in  to make it cleaner (these live in `PageForm/fields/validate`

I did this to make these fields reusable for other forms: see
- `front/app/modules/commercial/customizable_navbar/admin/components/NavbarItemForm.tsx`
- `front/app/modules/commercial/customizable_navbar/admin/components/PageFormWithNavbarNameField.tsx`

Reply:
Nice work ! 
I like the approach, my only concern is if these components are to be re-used, they might not be in the best place with the best name to be used by any other dev, maybe put them in the components folder, or the library, or simply communicate about them. Same for the validation functions, even more importantly I'd say.